From 01b045efca3d43ceb45d5e4a18a746ed643b69cd Mon Sep 17 00:00:00 2001
From: Yang Li <yang.lee@linux.alibaba.com>
Date: Wed, 20 Dec 2023 08:51:43 +0800
Subject: [PATCH 0977/1141] tools/counter: Remove unneeded semicolon

./tools/counter/counter_watch_events.c:233:3-4: Unneeded semicolon
./tools/counter/counter_watch_events.c:234:2-3: Unneeded semicolon
./tools/counter/counter_watch_events.c:333:2-3: Unneeded semicolon

Reported-by: Abaci Robot <abaci@linux.alibaba.com>
Closes: https://bugzilla.openanolis.cn/show_bug.cgi?id=7782
Signed-off-by: Yang Li <yang.lee@linux.alibaba.com>
Reviewed-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Link: https://lore.kernel.org/r/20231220005143.84987-1-yang.lee@linux.alibaba.com
Signed-off-by: William Breathitt Gray <william.gray@linaro.org>
(cherry picked from commit b7760cf94d4f2665bf40d08dd69aa5d0b4aa593f)
Change-Id: Ie05f19402a8abb126c9cce604b416b481a5cdc69
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/365869
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 tools/counter/counter_watch_events.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/tools/counter/counter_watch_events.c
+++ b/tools/counter/counter_watch_events.c
@@ -230,8 +230,8 @@ int main(int argc, char **argv)
 			break;
 		default:
 			return EXIT_FAILURE;
-		};
-	};
+		}
+	}
 
 	if (nwatch) {
 		watches = calloc(nwatch, sizeof(*watches));
@@ -330,7 +330,7 @@ int main(int argc, char **argv)
 			i++;
 			break;
 		}
-	};
+	}
 
 	if (debug)
 		print_watch(watches, nwatch);
